core-ui/src/components/Predefined/Details/Namespace/ResourcesUsage.js

```diff
@@ -0,0 +1,139 @@
+import React from 'react';
+
+import { Panel, LayoutGrid, Icon } from 'fundamental-react';
+import { useGetList, Spinner, CircleProgress } from 'react-shared';
+
+const MEMORY_SUFFIX_POWER = {
+  // must be sorted from the smallest to the largest; it is case sensitive; more info: https://medium.com/swlh/understanding-kubernetes-resource-cpu-and-memory-units-30284b3cc866
+  m: 1e-3,
+  K: 1e3,
+  Ki: 2 ** 10,
+  M: 1e6,
+  Mi: 2 ** 20,
+  G: 1e9,
+  Gi: 2 ** 30,
+  Ti: 2 ** 40,
+};
+
+function getBytes(memoryString) {
+  const suffixMatch = memoryString.match(/\D+$/);
+
+  if (!suffixMatch?.length) {
+    console.warn('error');
+    return;
+  }
+  const suffix = suffixMatch[0];
+  const number = memoryString.replace(suffix, '');
+
+  const suffixPower = MEMORY_SUFFIX_POWER[suffix];
+  if (!suffixPower) {
+    console.warn('error');
+    return;
+  }
+
+  return number * suffixPower;
+}
+
+function unitToPascalCase(str) {
+  const firstLetter = str[0];
+  return firstLetter.toUpperCase() + str.toLowerCase().slice(1);
+}
+
+function bytesToHumanReadable(bytesNumber) {
+  let output = bytesNumber;
+  Object.entries(MEMORY_SUFFIX_POWER).forEach(([suffix, power]) => {
+    const value = bytesNumber / power;
+    if (value >= 1)
+      output =
+        Math.round((value + Number.EPSILON) * 100) / 100 +
```

Comment:
value.toFixed(2)?

Reply:
We don't want to see a value like "2.00Gi/3.10Gi" instead of "2Gi/3.1Gi", do we?